backend/src/junior/junior.service.ts

```diff
@@ -131,15 +136,24 @@ export class JuniorService {
         user.birthday = details.birthday;
         user.parentsName = details.parentsName;
         user.parentsPhoneNumber = details.parentsPhoneNumber;
+        user.school = details.school;
+        user.class = details.class;
         user.postCode = details.postCode;
         user.homeYouthClub = details.homeYouthClub;
         user.gender = details.gender;
         user.nickName = details.nickName;
+        user.status = details.status;
+        user.photoPermission = details.photoPermission;
         const errors = await validate(user);
         if (errors.length > 0) {
             throw new BadRequestException(errors);
         }
         await this.juniorRepo.save(user);
+        if (prevStatus === 'p' && details.status === 'a') {
```

Comment:
What this line exactly does?

Reply:
p is pending and a is accepted. I guess I shouldn't have mapped them to one-letter variables as has been done for gender, I'll change it back to normal words then